tests/common/gcp_api/crm_test.py

```diff
@@ -169,6 +169,57 @@ def test_get_organization(self):
         self.crm_api_client.get_organization(org_name)
         self.assertEquals(1, crm.LOGGER.error.call_count)
 
+    def test_get_organizations(self):
+        """Test get organizations."""
+
+        mock_orgs_stub = mock.MagicMock()
+        self.crm_api_client.service = mock.MagicMock()
+        self.crm_api_client.service.organizations.return_value = mock_orgs_stub
+        
+        fake_orgs = {
```

Comment:
nit: can we move this to the test data files?  1: to be consistent, 2: to avoid clutter in the test code here?